docs/1_patching-applications.md

```diff
@@ -0,0 +1,27 @@
+# 🧩 Patching applications
+
```

Comment:
Some pages contain text under the title, some don't, please restore consistency in that regard.